apps/studio/components/layouts/ReportsLayout/ReportsLayout.tsx

```diff
@@ -20,7 +20,8 @@ const ReportsLayout = ({ title, children }: PropsWithChildren<ReportsLayoutProps
 
   const menuItems = [
     {
-      title: '',
+      title: 'Custom reports',
+      key: 'custom-reports',
```

Comment:
```suggestion
      key: 'reports',
```
Not sure if this is the fix, but this menu item is missing the active state